tox.ini

```diff
@@ -7,13 +7,11 @@
 envlist = py27
 
 [pytest]
-addopts = --cov=python_etl tests/
+addopts = --cov=mozetl tests/
 
 [testenv]
 commands = pytest
-passenv = PYTHONPATH
```

Comment:
Do tests pass for you without passing the PYTHONPATH? Tox is complaining about not having access to pyspark.sql.